engine/org.eclipse.birt.report.engine.emitter.html/src/org/eclipse/birt/report/engine/emitter/html/HTMLPerformanceOptimize.java

```diff
@@ -558,8 +559,12 @@ public void buildImageStyle(IImageContent image, StringBuffer styleBuffer, int d
 		boolean useHeightAutoValue = false;
 
 		// column width
-		DimensionType columnWidth = ((ICellContent) image.getParent()).getColumnInstance().getWidth();
-		if (image.isFitToContainer() && columnWidth != null) {
+		DimensionType columnWidth = null;
+		IElement parent = image.getParent();
+		if (parent != null && parent instanceof ICellContent) {
+			columnWidth = ((ICellContent) image.getParent()).getColumnInstance().getWidth();
```

Comment:
```suggestion
			columnWidth = ((ICellContent)parent).getColumnInstance().getWidth();
```
Best to use the variable.